dictionaries/CallMap.php

```diff
@@ -13471,11 +13471,11 @@
 'SplTempFileObject::valid' => ['bool'],
 'SplType::__construct' => ['void', 'initial_value='=>'mixed', 'strict='=>'bool'],
 'Spoofchecker::__construct' => ['void'],
-'Spoofchecker::areConfusable' => ['bool', 's1'=>'string', 's2'=>'string', '&w_error='=>'string'],
-'Spoofchecker::isSuspicious' => ['bool', 'text'=>'string', '&w_error='=>'string'],
-'Spoofchecker::setAllowedLocales' => ['void', 'locale_list'=>'string'],
-'Spoofchecker::setChecks' => ['void', 'checks'=>'long'],
-'Spoofchecker::setRestrictionLevel' => ['void', 'restriction_level'=>'int'],
+'Spoofchecker::areConfusable' => ['bool', 'string1'=>'string', 'string2'=>'string', '&w_errorCode='=>'int'],
+'Spoofchecker::isSuspicious' => ['bool', 'string'=>'string', '&w_errorCode='=>'int'],
```

Comment:
the name is `errorCode` in the PHP stubs, not `w_errorCode`

Reply:
This is an internal psalm annotation. The callmap is verified against reflection so it matches PHP 8.1 reflection.